tests/Fixtures/Criteria/Males.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Tests\Fixtures\Criteria;
+
+use EthicalJobs\SDK\Storage\Contracts\Criteria;
+use EthicalJobs\SDK\Storage\Contracts\Repository;
+
+class Males implements Criteria
+{
+    public function apply(Repository $repository)
+    {
+        $repository
+            ->where('sex', '=', 'male');
```

Comment:
Hmm. This field was actually defined as an _integer_ in the migration above, not a string.

I'm not sure what any of this code is even for, since sdk-php doesn't theoretically care about our users' genders? But it should be consistent regardless.

Reply:
The tests do pass, and this code is only used for tests, so I have to assume it's okay the way it is. 🤷‍♀️